fastlane/fastlane.gemspec

```diff
@@ -65,4 +65,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'appium_lib', '~> 4.1.0'
   spec.add_development_dependency 'rest-client', '~> 1.6.7'
   spec.add_development_dependency 'fakefs', '~> 0.8.1'
+  spec.add_development_dependency 'xcode-install'
```

Comment:
Should be fine now, since there's no longer any pinning on an older version of `spaceship` present.

![](http://i.imgur.com/c4jt321.png)